optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/UseCasesAndExamples-chapter.adoc

```diff
@@ -244,34 +242,35 @@ These realistic competitions provide an objective comparison of Planner with com
 [[nQueensProblemDescription]]
 ==== Problem Description
 
-Place n queens on a n sized chessboard so no 2 queens can attack each other.
-The most common n queens puzzle is the 8 queens puzzle, with __n = 8__:
+Place _n_ queens on a _n_ sized chessboard so that no two queens can attack each other.
+The most common _n_ queens puzzle is the eight queens puzzle, with __n = 8__:
 
 image::UseCasesAndExamples/nQueensScreenshot.png[align="center"]
 
 Constraints:
 
 * Use a chessboard of _n_ columns and _n_ rows.
 * Place _n_ queens on the chessboard.
-* No 2 queens can attack each other. A queen can attack any other queen on the same horizontal, vertical or diagonal line.
+* No two queens can attack each other. A queen can attack any other queen on the same horizontal, vertical or diagonal line.
 
-This documentation heavily uses the 4 queens puzzle as the primary example.
+This documentation heavily uses the four queens puzzle as the primary example.
```

Comment:
Yes, you're right with this change (as wikipedia says "eight queens puzzle" and not "8 queens puzzle").
In the entire manual, I regularly use the phrase "In 8 queens, <statement>" or "in the cloud balancing example, <statement>" etc. Those will need to be consistently fixed. Some proposals:
- In the eight queens example, <statement>
- In the eight queens puzzle, <statement>
- In eight queens, <statement>
Of course, it's often "four" or "n" instead of "eight".
I am fine with whatever proposal you prefer, as long as it's consistently used in the manual.

Because it's easy to find these with "find in path" (which finds all occurrences of a text in all files in a folder), I suggest to fix these first as a single PR, before reviewing the other chapters.

@gsheldon Wdyt?